app/mailers/hypertension_fast_track_pilot_mailer.rb

```diff
@@ -3,7 +3,8 @@
 class HypertensionFastTrackPilotMailer < ApplicationMailer
   def build(submission)
     @submission = submission
-    @disability = 'hypertension'
+    @disability = submission.form.dig('form526', 'form526', 'disabilities')&.first
+    @disability_code = @disability['diagnosticCode'] if @disability
```

Comment:
I think the safe-nav syntax for array access is `@disability&.[](:diagnosticCode)` -- not the prettiest.